### ISSUE TITLE
Zamtrios LR and HR Armor Helmet Bad Texture

### ISSUE THREAD

**Issue by Manfred60** [Monkbreh/MHGU-HD-Project#7]
The Zamtrios helmet looks like a block of ice, the visor and other details are not upscaled correctly.

**Comment by Monkbreh** [comment]
Yes already fixed that, uhh going to release it very soon. Am just identifying other incorrect transparency problems .

**Comment by Manfred60** [comment]
Okay that's great to hear! I am looking forward to the next release!
